packages/table/components/Table.tsx

```diff
@@ -152,16 +171,35 @@ export class Table<T> extends React.PureComponent<TableProps, TableState> {
     }
   ) {
     const { key, style, column } = args;
-    const className = css(headerCss, {
-      ...style
-    });
     return (
-      <div className={className} key={key}>
+      <div
+        className={headerCss}
+        style={{
+          ...style
```

Comment:
💅 No need in spreading `style={style}`, am I missing something?

Reply:
Why is it better than `emotion.css`?

Reply:
@nLight - if we don't put it in the style attribute, react-virtualized sets `width: auto` in the style attribute which overrides whatever we add using `emotion`